src/Raven.Client/Documents/Queries/IndexQuery.cs

```diff
@@ -125,11 +127,13 @@ public abstract class IndexQueryBase<T> : IIndexQuery, IEquatable<IndexQueryBase
         /// <summary>
         /// Number of records that should be skipped.
         /// </summary>
+        [Obsolete("Use OFFSET in RQL instead")]
```

Comment:
Why mark it `Obsolete`? You can either specify this in the query or outside, both will work fine.
The smaller of the values will take affect

Reply:
Why, we always want to be able to narrow down the paging without touching the actual query

Reply:
This is why I'm using 'query parameters' for paging in `AbstractDocumentQuery.ToString()`

Reply:
I don't follow?